src/Core/Extensions/MessageExtensions.cs

```diff
@@ -14,22 +14,40 @@ public static async Task SendToChannel(this Embed e, IMessageChannel channel)
             => await channel.SendMessageAsync(string.Empty, false, e);
         public static EmbedBuilder AddFieldConditional(this EmbedBuilder eb, bool condition, string name, object value, bool inline = false)
         {
-            if (condition)
-                eb.AddField(name, value, inline);
+            if (condition) {
+                if (value is string) {
+                    eb.AddField(name, cropToLength(value as string, 1024), inline);
```

Comment:
The `EmbedBuilder` class has the constants for maximum field length, maybe pull it from there in case the API is changed.

Reply:
Apparently not for 1024, I made it descriptionlength / 2.

Reply:
`EmbedFieldBuilder.MaxFieldValueLength` is what we are looking for, sorry!